tiny_dnn/tiny_dnn.h

```diff
@@ -84,6 +86,13 @@ using ave_pool = tiny_dnn::average_pooling_layer;
 
 using fc = tiny_dnn::fully_connected_layer;
 
+// TODO(prlz77) RNN Wrapper
+// template<class T>
+// using recurrent = tiny_dnn::recurrent_layer<T>;
+
+template <class activation_type>
+using recurrent_cell = tiny_dnn::recurrent_cell_layer<activation_type>;
```

Comment:
I mean, the activation is just a small part of the rnn formula, it is not applied afterwards, such as after a convolution or a fully connected.